app/services/worlds_service.py

```
@@ -0,0 +1,91 @@
+
+    def __init__(self, repository: WorldsRepository, gw2_client: GW2Client):
+        self.repository = repository
+        self.gw2_client = gw2_client
+
+    async def get_all_worlds(self) -> list[dict]:
+        """
+        Get all worlds from database. If DB is empty, fetch from GW2 API and sync with database in the background.
+        """
+        try:
+            worlds_data = await self._get_worlds_from_db()
+            return worlds_data
```

Ruff (0.14.1)

Consider moving this statement to an `else` block

(TRY300)